### DISCUSSION TITLE
Ranked beatmap's hitsounds are out of sync with the song

### DISCUSSION THREAD

**Discussion by Crozh** [ppy/osu#22637] [Q&A]
### Type

Game behaviour

### Bug description

I've been always amazed for how strange this is, but this beatmapset in general ( https://osu.ppy.sh/beatmapsets/1497801#osu/3072846 ) Is always out of sync. This has been an "issue" for almost a year now, I've tried playing it on different OSes, different clients (meaning live and lazer) but the hitsounds are **always** out of sync with the song and I don't know why. 
This probably is an issue with my setup or something? 'cause this doesn't happen with any other map besides this one; and yes, I've tried downloading it again and it is the same. 
Offset is at 0, using Linux and pipewire sound server. I'm using an E10k with Audio Technica M50x headphones. 
The change in skins was because of the recording hotkey being the same as the hotkey for changing skins, lmao (CTRL+SHIFT+R). However, this doesn't change the behavior I'm presenting.
The logs were from a session in which I completed the beatmap at fault, just in case.  

### Screenshots or videos

https://user-images.githubusercontent.com/44332979/218654369-62f98437-d366-4352-a94f-4b3615e77ef3.mp4

https://user-images.githubusercontent.com/44332979/218654389-37070398-a38b-4359-85f5-ede853435d21.mp4

### Version

2023.207.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/10729229/runtime.log)
[input.log](https://github.com/ppy/osu/files/10729231/input.log)
[network.log](https://github.com/ppy/osu/files/10729232/network.log)
[performance.log](https://github.com/ppy/osu/files/10729233/performance.log)
[database.log](https://github.com/ppy/osu/files/10729234/database.log)

**Comment by peppy** [comment]
Cannot reproduce FWIW.

**Reply by Crozh** [reply]
So no way of a solution, I guess.
